content/docs/user-guide/project-config/cli-reference.md

```diff
@@ -2121,3 +2122,137 @@ o3de.bat sha256 --file-path FILE_PATH --json-path JSON_PATH
 - **`-j JSON_PATH, --json-path JSON_PATH`**
 
   The path to the O3DE object's JSON file that you want to add the `sha256` hash value to.
+
+<!-------------------------------------------------------------->
+
+## `android-configure`
+
+Manages the settings used by the Android Project Generation process to create an Android Gradle build script.
+
+### Format
+
+```cmd
+android-configure [-h] [--global] [-p PROJECT] [-l] [--validate] [--set-value VALUE] [--clear-value VALUE] [--set-password SETTING] [--debug]
+```
+
+### Usage
+
+```cmd
+android-configure -l
+```
+This will list the current settings that the Android Project Generation script will use.
+
+```cmd
+android-configure --validate
+```
+This will perform a dry run and validate that the minimal settings and configured and the prequisite software environments are satisfied. If not, it will report the detailed error.
```

Comment:
```suggestion
This will perform a dry run and validate that the minimal settings are configured and the prerequisite software environments are satisfied. If not, it will report a detailed error.
```